components/teams-service/storage/postgres/migration/sql/08_update_admin_team_name.up.sql

```diff
@@ -0,0 +1,9 @@
+BEGIN;
+
+UPDATE teams
+    SET
+        description = 'admins'
+    WHERE
+        name = 'admins';
```

Comment:
❓ This will affect IAMv1, and we're ok with that. Just stating the obvious 😉

Reply:
yup we quadrupled checked with @susanev 😂